.github/workflows/docker-image.yml

```
@@ -4,296 +4,283 @@ on:
+            "
+
+      - name: Test FPM process and configuration
         run: |
-          # Create manifest list
-          docker manifest create ${{ steps.read-tags.outputs.base-tag }} \
-            ${{ steps.read-tags.outputs.linux-tag }} \
-            ${{ steps.read-tags.outputs.mac-tag }}
+          echo "Testing PHP-FPM process..."
+          CONTAINER_ID=$(docker run -d --platform ${{ matrix.platform }} \
+            ${{ steps.image-ref.outputs.image-ref }})
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)